doc/content/source/filters/EnergyOutFilter.md

```diff
@@ -32,12 +32,12 @@ scattered particles into two groups using the CASMO-2 group structure.
     [Heating]
       type = CellTally
       score = 'kappa_fission'
-      blocks = '100 200'
```

Comment:
Can we use a file link instead of duplicating input here? This will insulate us from other syntax changes.